python/tidy.py

```diff
@@ -7,17 +7,17 @@
 # option. This file may not be copied, modified, or distributed
 # except according to those terms.
 
-#!/usr/bin/env python
-
 import os
 import fnmatch
 import itertools
 import re
+import sys
 from licenseck import licenses
 
 filetypes_to_check = [".rs", ".rc", ".cpp", ".c", ".h", ".py"]
 reftest_directories = ["tests/ref"]
 reftest_filetype = ".list"
+dependencies = ["./python/flake8-2.4.1-py2.py3-none-any.whl"]
```

Comment:
If anyone has a better suggestion for the location of this wheel, let me know. I just kind of threw it in there thinking someone might have a better idea where it should live